app/src/main/java/ch/epfl/sweng/rps/ui/friends/FriendsInfo.kt

```diff
@@ -0,0 +1,12 @@
+package ch.epfl.sweng.rps.ui.friends
+
+data class FriendsInfo(
+    val username : String,
+    val gamesPlayed : Int,
+    val gamesWon : Int,
```

Comment:
It seems a bit weird to me to have this class here. But I am not sure. Maybe this could be in models??

Reply:
You're right, let me move that.

Reply:
I will also work on changing to UI in the next sprint to match the rest of the application. Didn't have enough time for that this week